examples/postgresql-psc/main.tf

```
@@ -25,6 +25,22 @@ locals {
   }
 }
 
+module "network-auto-psc" {
+  source  = "terraform-google-modules/network/google"
+  version = "~> 9.0"
+
+  project_id   = var.project_id
+  network_name = "your_network_name"
```

Looks like test failure was due to name constraints ` Returning due to fatal error: FatalError{Underlying: error while running command: exit status 1; 
Error: "name" ("your_network_name") doesn't match regexp "^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$"`

```suggestion
  network_name = "sample-network"
```